UniWatch/Views/Lecture/Delete.cshtml

```diff
@@ -0,0 +1,16 @@
+﻿@model UniWatch.ViewModels.Lecture.LectureViewModel
+@{
+    ViewBag.Title = "Delete";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<h2>Delete</h2>
```

Comment:
Which lecture is being deleted? User will have no context. Missing prompt "... cannot be undone"